tabml/data_processing.py

```diff
@@ -68,12 +68,12 @@ def __init__(self, whisker_coeff: float = 1.5):
         self.lower: float = float("-inf")
         self.upper: float = float("inf")
 
-    def fit(self, X: pd.Series):
+    def fit(self, X: pd.DataFrame, y=None):
```

Comment:
`fit` requires 2 arguments (other than `self`) to be compatible with sklearn Pipeline.